Index: slime/doc/Makefile
===================================================================
--- slime.orig/doc/Makefile	2012-04-24 21:18:46.000000001 +0200
+++ slime/doc/Makefile	2012-04-24 21:23:47.000000001 +0200
@@ -7,6 +7,7 @@
 # installed their emacs from source probably have a GCS-ish file
 # hierarchy.
 infodir=/usr/local/info
+HOME=$(CURDIR)
 
 # What command to use to install info file(s)
 INSTALL_CMD=install -m 644
@@ -16,10 +17,10 @@
 
 TEXI = slime.texi contributors.texi
 
-all: slime.info slime.pdf html/index.html
+all: slime.ps slime.info slime.html slime.pdf html/index.html slime-refcard.pdf
 
 slime.dvi: $(TEXI)
-	texi2dvi slime.texi
+	HOME=$(HOME) texi2dvi slime.texi
 
 slime.ps: slime.dvi
 	dvips -o $@ $<
@@ -43,7 +44,10 @@
 	ssh $(CLUSER)@common-lisp.net "cd $(DOCDIR); tar -zxf html.tgz"
 
 slime.pdf: $(TEXI)
-	texi2pdf $<
+	HOME=$(HOME) texi2pdf $<
+
+slime-refcard.pdf: slime-refcard.tex
+	HOME=$(HOME) texi2pdf $<	
 
 install: install-info
 
@@ -99,10 +103,11 @@
 		install-info --infodir="$(infodir)" --remove "$(infodir)/$(infofiles)" || :; fi
 	rm -f $(infodir)/$(infofiles)
 
+SHELL=/bin/bash
+
 clean:
 	rm -f contributors.texi
-	rm -f slime.aux slime.cp slime.cps slime.fn slime.fns slime.ky
-	rm -f slime.kys slime.log slime.pg slime.tmp slime.toc slime.tp
-	rm -f slime.vr slime.vrs
-	rm -f slime.info slime.pdf slime.dvi slime.ps slime.html
-	rm -rf html html.tgz
+	rm -f slime.{aux,cp,cps,fn,fns,ky,kys,log,pg,tmp,toc,tp,vr,vrs}
+	rm -f slime.{info,pdf,dvi,ps,html,kys,fns,vrs}
+	rm -f slime-refcard.{aux,log,pdf}
+	rm -rf html{,.tgz} .texmf-var
