Index: slime/doc/Makefile
===================================================================
--- slime.orig/doc/Makefile	2010-02-20 18:55:59.000000000 +0100
+++ slime/doc/Makefile	2010-02-20 18:56:07.000000000 +0100
@@ -16,7 +16,7 @@
 
 TEXI = slime.texi contributors.texi
 
-all: slime.info slime.pdf html/index.html
+all: slime.ps slime.info slime.html slime.pdf html/index.html slime-refcard.pdf
 
 slime.dvi: $(TEXI)
 	texi2dvi slime.texi
@@ -45,6 +45,9 @@
 slime.pdf: $(TEXI)
 	texi2pdf $<
 
+slime-refcard.pdf: slime-refcard.tex
+	texi2pdf $<	
+
 install: install-info
 
 uninstall: uninstall-info
@@ -99,10 +102,11 @@
 		install-info --infodir="$(infodir)" --remove "$(infodir)/$(infofiles)" || :; fi
 	rm -f $(infodir)/$(infofiles)
 
+SHELL=/bin/bash
+
 clean:
 	rm -f contributors.texi
-	rm -f slime.aux slime.cp slime.cps slime.fn slime.fns slime.ky
-	rm -f slime.kys slime.log slime.pg slime.tmp slime.toc slime.tp
-	rm -f slime.vr slime.vrs
-	rm -f slime.info slime.pdf slime.dvi slime.ps slime.html
-	rm -rf html html.tgz
+	rm -f slime.{aux,cp,cps,fn,fns,ky,kys,log,pg,tmp,toc,tp,vr,vrs}
+	rm -f slime.{info,pdf,dvi,ps,html,kys,fns,vrs}
+	rm -f slime-refcard.{aux,log,pdf}
+	rm -rf html{,.tgz}
