Index: slime/slime.el
===================================================================
--- slime.orig/slime.el	2013-04-18 22:55:58.000000001 +0200
+++ slime/slime.el	2013-04-18 22:57:37.000000001 +0200
@@ -9466,7 +9466,7 @@
 (run-hooks 'slime-load-hook)
 
 ;; Local Variables:
-;; lexical-binding: t
+;; lexical-binding: nil
 ;; outline-regexp: ";;;;+"
 ;; indent-tabs-mode: nil
 ;; coding: latin-1-unix
