Index: slime/slime.el
===================================================================
--- slime.orig/slime.el	2012-07-01 20:59:26.000000001 +0200
+++ slime/slime.el	2012-07-01 21:00:50.000000001 +0200
@@ -9380,7 +9380,7 @@
 (run-hooks 'slime-load-hook)
 
 ;; Local Variables: 
-;; lexical-binding: t
+;; lexical-binding: nil
 ;; outline-regexp: ";;;;+"
 ;; indent-tabs-mode: nil
 ;; coding: latin-1-unix
