--- a/slime.el
+++ b/slime.el
@@ -141,6 +141,15 @@
 (defvar slime-protocol-version nil)
 (setq slime-protocol-version slime-version)
 
+(defun slime-gather-lisp-implementations ()
+  "Compute sane default value for slime-lisp-implementations."
+  (let ((known-lisps '("sbcl" "clisp" "ecl" "cmucl")))
+    (cl-remove-if
+     'null
+     (mapcar 
+      (lambda (l) (if (executable-find l) (list l (list l))))
+      known-lisps))))
+
 
 ;;;; Customize groups
 ;;
@@ -915,7 +924,7 @@
 (defvar inferior-lisp-program "lisp"
   "*Program name for invoking an inferior Lisp with for Inferior Lisp mode.")
 
-(defvar slime-lisp-implementations nil
+(defvar slime-lisp-implementations (slime-gather-lisp-implementations)
   "*A list of known Lisp implementations.
 The list should have the form:
   ((NAME (PROGRAM PROGRAM-ARGS...) &key KEYWORD-ARGS) ...)
