Index: slime/swank-loader.lisp
===================================================================
--- slime.orig/swank-loader.lisp	2014-10-21 20:14:10.000000001 +0200
+++ slime/swank-loader.lisp	2014-10-21 20:17:08.000000001 +0200
@@ -41,7 +41,7 @@
   #+clozure '(metering (swank ccl) (swank gray))
   #+lispworks '((swank lispworks) (swank gray))
   #+allegro '((swank allegro) (swank gray))
-  #+clisp '(xref metering (swank clisp) (swank gray))
+  #+clisp '(#+() xref metering (swank clisp) (swank gray))
   #+armedbear '((swank abcl))
   #+cormanlisp '((swank corman) (swank gray))
   #+ecl '((swank ecl) (swank gray))
