Index: slime/swank-clisp.lisp
===================================================================
--- slime.orig/swank-clisp.lisp	2010-07-22 10:33:13.000000000 +0200
+++ slime/swank-clisp.lisp	2010-07-22 10:36:44.000000000 +0200
@@ -631,6 +631,7 @@
 
 ;;;; Portable XREF from the CMU AI repository.
 
+#| PVE remove xref
 (setq pxref::*handle-package-forms* '(cl:in-package))
 
 (defmacro defxref (name function)
@@ -643,6 +644,7 @@
 (defxref who-sets       pxref:list-setters)
 (defxref list-callers   pxref:list-callers)
 (defxref list-callees   pxref:list-callees)
+|#
 
 (defun xref-results (symbols)
   (let ((xrefs '()))
Index: slime/swank-loader.lisp
===================================================================
--- slime.orig/swank-loader.lisp	2010-07-22 10:36:08.000000000 +0200
+++ slime/swank-loader.lisp	2010-07-22 10:36:58.000000000 +0200
@@ -40,7 +40,7 @@
   #+clozure '(metering swank-ccl swank-gray)
   #+lispworks '(swank-lispworks swank-gray)
   #+allegro '(swank-allegro swank-gray)
-  #+clisp '(xref metering swank-clisp swank-gray)
+  #+clisp '(#+() xref metering swank-clisp swank-gray)
   #+armedbear '(swank-abcl)
   #+cormanlisp '(swank-corman swank-gray)
   #+ecl '(swank-source-path-parser swank-source-file-cache
