Index: slime/swank-clisp.lisp
===================================================================
--- slime.orig/swank-clisp.lisp	2012-04-21 21:41:50.000000001 +0200
+++ slime/swank-clisp.lisp	2012-04-21 21:47:09.000000001 +0200
@@ -692,6 +692,7 @@
 
 ;;;; Portable XREF from the CMU AI repository.
 
+#| PVE remove xref
 (setq pxref::*handle-package-forms* '(cl:in-package))
 
 (defmacro defxref (name function)
@@ -704,6 +705,7 @@
 (defxref who-sets       pxref:list-setters)
 (defxref list-callers   pxref:list-callers)
 (defxref list-callees   pxref:list-callees)
+|#
 
 (defun xref-results (symbols)
   (let ((xrefs '()))
Index: slime/swank-loader.lisp
===================================================================
--- slime.orig/swank-loader.lisp	2012-04-21 21:41:50.000000001 +0200
+++ slime/swank-loader.lisp	2012-04-21 21:47:09.000000001 +0200
@@ -39,7 +39,7 @@
   #+clozure '(metering swank-ccl swank-gray)
   #+lispworks '(swank-lispworks swank-gray)
   #+allegro '(swank-allegro swank-gray)
-  #+clisp '(xref metering swank-clisp swank-gray)
+  #+clisp '(#+() xref metering swank-clisp swank-gray)
   #+armedbear '(swank-abcl)
   #+cormanlisp '(swank-corman swank-gray)
   #+ecl '(swank-source-path-parser swank-source-file-cache
