--- a/swank-loader.lisp
+++ b/swank-loader.lisp
@@ -17,6 +17,10 @@
 ;;   (setq swank-loader::*fasl-directory* "/tmp/fasl/")
 ;;   (swank-loader:init)
 
+
+(handler-case (require :asdf)
+  (error () (load "/usr/share/common-lisp/source/cl-asdf/asdf.lisp")))
+
 (cl:defpackage :swank-loader
   (:use :cl)
   (:export :init
@@ -139,7 +143,7 @@
 (defun slime-version-string ()
   "Return a string identifying the SLIME version.
 Return nil if nothing appropriate is available."
-  (with-open-file (s (merge-pathnames "slime.el" *source-directory*)
+  (with-open-file (s "/usr/share/doc/cl-swank/changelog"
                      :if-does-not-exist nil)
     (loop with prefix = ";; Version: "
           for line = (read-line s nil :eof)
