Index: slime/slime.el
===================================================================
--- slime.orig/slime.el	2010-07-22 10:36:44.000000000 +0200
+++ slime/slime.el	2010-07-22 10:36:45.000000000 +0200
@@ -118,7 +118,7 @@
     "Return the datestring of the latest entry in the ChangeLog file.
 Return nil if the ChangeLog file cannot be found."
     (interactive "p")
-    (let ((changelog (concat slime-path "ChangeLog"))
+    (let ((changelog "/usr/share/doc/cl-swank/changelog")
           (date nil))
       (when (file-exists-p changelog)
         (with-temp-buffer 
Index: slime/swank-loader.lisp
===================================================================
--- slime.orig/swank-loader.lisp	2010-07-22 10:36:44.000000000 +0200
+++ slime/swank-loader.lisp	2010-07-22 10:36:45.000000000 +0200
@@ -119,7 +119,7 @@
 (defun slime-version-string ()
   "Return a string identifying the SLIME version.
 Return nil if nothing appropriate is available."
-  (with-open-file (s (merge-pathnames "ChangeLog" *source-directory*)
+  (with-open-file (s "/usr/share/doc/cl-swank/changelog" 
                      :if-does-not-exist nil)
     (and s (symbol-name (read s)))))
 
@@ -136,10 +136,11 @@
 
 (defun binary-pathname (src-pathname binary-dir)
   "Return the pathname where SRC-PATHNAME's binary should be compiled."
-  (let ((cfp (compile-file-pathname src-pathname)))
-    (merge-pathnames (make-pathname :name (pathname-name cfp)
-                                    :type (pathname-type cfp))
-                     binary-dir)))
+  (declare (ignore binary-dir))
+  (let ((cfp (compile-file-pathname src-pathname)))    
+    (merge-pathnames (make-pathname 
+                      :directory `(:relative "swank" "fasl" ,(unique-dir-name)))
+                     (asdf:apply-output-translations cfp))))
 
 (defun handle-swank-load-error (condition context pathname)
   (fresh-line *error-output*)
