Index: slime/slime.el
===================================================================
--- slime.orig/slime.el	2015-04-28 21:15:08.000000000 +0200
+++ slime/slime.el	2015-04-28 21:17:12.438427583 +0200
@@ -128,11 +128,7 @@
 
 (eval-and-compile
   (defun slime--changelog-file-name ()
-    (expand-file-name "ChangeLog"
-                      (if (and (boundp 'byte-compile-current-file)
-                               byte-compile-current-file)
-                          (file-name-directory byte-compile-current-file)
-                          slime-path)))
+    "/usr/share/doc/slime/changelog")
 
   (defun slime-changelog-date (&optional interactivep)
     "Return the datestring of the latest entry in the ChangeLog file.
Index: slime/swank-loader.lisp
===================================================================
--- slime.orig/swank-loader.lisp	2015-04-28 21:15:08.000000000 +0200
+++ slime/swank-loader.lisp	2015-04-28 21:15:58.095111832 +0200
@@ -17,6 +17,10 @@
 ;;   (setq swank-loader::*fasl-directory* "/tmp/fasl/")
 ;;   (swank-loader:init)
 
+
+(handler-case (require :asdf)
+  (error () (load "/usr/share/common-lisp/source/cl-asdf/asdf.lisp")))
+
 (cl:defpackage :swank-loader
   (:use :cl)
   (:export :init
@@ -134,7 +138,7 @@
 (defun slime-version-string ()
   "Return a string identifying the SLIME version.
 Return nil if nothing appropriate is available."
-  (with-open-file (s (merge-pathnames "ChangeLog" *source-directory*)
+  (with-open-file (s "/usr/share/doc/cl-swank/changelog" 
                      :if-does-not-exist nil)
     (and s (symbol-name (read s)))))
 
@@ -151,10 +155,11 @@
 
 (defun binary-pathname (src-pathname binary-dir)
   "Return the pathname where SRC-PATHNAME's binary should be compiled."
-  (let ((cfp (compile-file-pathname src-pathname)))
-    (merge-pathnames (make-pathname :name (pathname-name cfp)
-                                    :type (pathname-type cfp))
-                     binary-dir)))
+  (declare (ignore binary-dir))
+  (let ((cfp (compile-file-pathname src-pathname)))    
+    (merge-pathnames (make-pathname 
+                      :directory `(:relative "swank" "fasl" ,(unique-dir-name)))
+                     (asdf:apply-output-translations cfp))))
 
 (defun handle-swank-load-error (condition context pathname)
   (fresh-line *error-output*)
