Index: slime/slime-autoloads.el
===================================================================
--- slime.orig/slime-autoloads.el	2010-02-20 19:00:57.000000000 +0100
+++ slime/slime-autoloads.el	2010-02-20 19:01:04.000000000 +0100
@@ -39,6 +39,19 @@
   (setq slime-setup-contribs contribs)
   (add-hook 'slime-load-hook 'slime-setup-contribs))
 
+(defvar slime-setup-contribs nil)
+
+(defun slime-setup-contribs () 
+  (when slime-setup-contribs
+    (add-to-list 'load-path (expand-file-name "contrib" slime-path))
+    (dolist (c slime-setup-contribs)
+      (require c)
+      (let ((init (intern (format "%s-init" c))))
+        (when (fboundp init)
+          (funcall init))))))
+
+(slime-setup)
+
 (provide 'slime-autoloads)
 
 ;;; slime-autoloads.el ends here
