Index: slime/slime-autoloads.el
===================================================================
--- slime.orig/slime-autoloads.el	2013-04-18 22:47:43.000000001 +0200
+++ slime/slime-autoloads.el	2013-04-18 22:54:46.000000001 +0200
@@ -39,6 +39,18 @@
       (add-hook 'lisp-mode-hook 'slime-lisp-mode-hook))
     (setq slime-setup-contribs contribs)
     (add-hook 'slime-load-hook 'slime-setup-contribs))
+  
+  (defun slime-setup-contribs ()
+    "Load and initialize contribs."
+    (when slime-setup-contribs
+      (add-to-list 'load-path (expand-file-name "contrib" slime-path))
+      (dolist (c slime-setup-contribs)
+        (require c)
+        (let ((init (intern (format "%s-init" c))))
+          (when (fboundp init)
+            (funcall init))))))
+
+  (slime-setup)
 
   (provide 'slime-autoloads))
 
